workflows/batch_workflow.py

```diff
@@ -155,10 +155,13 @@ def main(  # pylint: disable=too-many-arguments,too-many-locals,too-many-stateme
     )
     scripts_dir = abspath(join(dirname(__file__), '..', 'scripts'))
 
-    analysis_base_bucket = (
-        f'gs://cpg-{callset_name}-{output_bucket_suffix}/joint-calling'
-    )
-    analysis_bucket = f'{analysis_base_bucket}/{callset_version}'
+    plots_bucket = f'gs://cpg-{callset_name}-{output_bucket_suffix}/joint-calling/{callset_version}'
+
+    if output_bucket_suffix in ['test', 'main']:
+        output_metadata_suffix = f'{output_bucket_suffix}-metadata'
+    else:
+        output_metadata_suffix = output_bucket_suffix
```

Comment:
When would this be used? Do we have another "testing" namespace apart from `test`?

Reply:
`test-tmp` :) Usually for test runs from a development branch.